cardano-db-sync/src/Cardano/DbSync/Plugin/Epoch.hs

```diff
@@ -137,8 +189,8 @@ latestCachedEpochVar = unsafePerformIO $ newIORef Nothing -- Gets updated later.
 latestChainTipEpochVar :: IORef Word64
 latestChainTipEpochVar = unsafePerformIO $ newIORef 0 -- Gets updated later.
 
-updateEpochNum :: MonadIO m => Word64 -> Trace IO Text -> ReaderT SqlBackend m (Either DbSyncNodeError ())
-updateEpochNum epochNum trce = do
+updateEpochNumDefault :: MonadIO m => Word64 -> Trace IO Text -> ReaderT SqlBackend m (Either DbSyncNodeError ())
```

Comment:
Why default?

Reply:
Because it's possible to change this default behavior.
I didn't add all the tests I tried, this was one of the things I tried to test :wink: